homeworks/yehor.honcharenko_NOcode-developer/GamesProject/Cards.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
```

Comment:
super, please rename these files to lowercase:
- Cards.html -> cards.html
- About.css -> about.css
- Cards.css -> cards.css
Maybe you should use `git rm` command, I'm not sure. Ask chatGPT or Bard. In your question add your OS, is it mac or linux

Reply:
At the end of your fixes, you should see the lowercase in the file names on GitHub
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/edb43ecc-bb94-4bbd-926f-09bb4faa1106)

Reply:
The Capital letters are still there. Proly you have to use `git rm` command, please consult with Google or bots

Reply:
> The Capital letters are still there. Proly you have to use `git rm` command, please consult with Google or bots

I executed the `git rm` command as suggested by Bard. Afterward, I used 'git add' and created a pull request with the changes in the names. I hope this will help resolve the issue because I don't see any other solutions besides canceling the pull request and creating it anew.

Reply:
well done 👏